Uni_life_tracker/app/src/main/res/layout/activity_main_page.xml

```diff
@@ -38,48 +51,41 @@
                 app:layout_constraintVertical_bias="0.025"
                 app:layout_constraintHorizontal_bias="0.05"/>
 
-        <!--  My To Do's  -->
-        <TextView
-                android:id="@+id/tv_to_dos"
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:text="@string/my_to_dos"
-                android:textSize="25sp"
-                android:fontFamily="@font/roboto_mono_bold"
-                android:textStyle="bold"
-                android:layout_marginTop="5dp"
-                android:layout_marginStart="20dp"/>
-        <!--  Welcome  -->
-        <TextView
-                android:id="@+id/tv_welcome"
-                android:layout_width="wrap_content"
+        <LinearLayout
+                android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:text="@string/welcome"
-                android:textSize="15sp"
-                android:fontFamily="@font/roboto_mono_light"
                 android:layout_marginTop="5dp"
-                android:layout_marginStart="20dp"/>
+                android:layout_marginHorizontal="20dp"
+                android:gravity="center_vertical">
 
-        <!--  Welcome Name  -->
-        <TextView
```

Comment:
Would it be possible to move up the welcome message just so the spacing isn't too tight between the date and the welcome text?

Reply:
yup, added spacing to it!